### PR TITLE
Bump lodash version to 4.17.15

### PR DESCRIPTION
**Pull request by bobwise**
## What does this PR do?
Update the dependency version for the react site, react components, and angular components. The Angular PeerDependency has been changed to a Dependency.

### Associated Issue 
Fixes https://github.com/sparkdesignsystem/spark-design-system/issues/1719.

### Code
 - [x] Unit Testing in Spark Vanilla with `npm run test` (100% coverage, 100% passing)
 - [x] Unit Testing in Spark Angular with `gulp test-angular` (100% coverage, 100% passing)
 - [x] Unit Testing in Spark React with `gulp test-react` (100% coverage, 100% passing)

### CODE REVIEW

File: src/angular/projects/spark-angular/package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "zone.js": "^0.8.26"
   },
   "dependencies": {
-    "lory.js": "^2.5.3"
+    "lory.js": "^2.5.3",
+    "lodash": "^4.17.15"
```
Comment (afebbraro) [inline src/angular/projects/spark-angular/package.json]: Did you mean to leave lodash as a devDependency and also add it as a dependency? Do we need both?

Comment (bobwise) [reply src/angular/projects/spark-angular/package.json]: lodash was added as a dependency and removed as a peerDependency. It should only appear once in this file.

Comment (bobwise) [reply src/angular/projects/spark-angular/package.json]: Does anything need to be changed here or can we resolve the review?

Comment (afebbraro) [reply src/angular/projects/spark-angular/package.json]: @bobwise, looks like you are changing the angular and spark versions to an old version here. Can you remove those changes? Them being '>=7.0.0 <=8.0.0' has been changed and the spark version has been updated too.

**Pull request merged**